src/lib-components/forms/DateRangePicker.vue

```diff
@@ -11,11 +11,13 @@ const props = withDefaults(
       minDate: number
       maxDate: number
     }
+    allowedRangeInDays?: number
```

Comment:
nit: I think I'd just go with `maxRange`, which feels intuitive enough for me, given the picker doesn't currently allow time selections.